plugin/cr/src/test/java/org/opencds/cqf/ruler/cr/r4/RepositoryServiceTest.java

```diff
@@ -1581,10 +1581,11 @@ void artifact_diff_compare_executable() {
 		Parameters grouperChanges = returnedParams.getParameter().stream().filter(p -> p.getName().contains("/dxtc")).map(p-> (Parameters)p.getResource()).findFirst().get();
 		List<ParametersParameterComponent> deleteOperations = getOperationsByType(grouperChanges.getParameter(), "delete");
 		List<ParametersParameterComponent> insertOperations = getOperationsByType(grouperChanges.getParameter(), "insert");
+		// TODO these are now 0, is this expected with removal of dao for expansion?
```

Comment:
hmm, no it should still be adding or removing the same number of codes. Before running the diff operation does the expanded ValueSet show the changes in expansion content?